app/src/main/java/com/github/sdp/tarjetakuna/MainActivity.kt

```diff
@@ -31,6 +34,30 @@ class MainActivity : AppCompatActivity() {
         }
     }
 
+    //To be removed
+    private val testCollection = listOf(
```

Comment:
I have to tell you that I don't agree. This is not a collection test, but this is what the collection export will permanently return, because the (local) database is not yet fully implemented. Once implemented, we will permanently take the database.

Reply:
Yes, i know the DB is not fully implemented, however, you should not test with the DB in any case when you export, but with a know collection test, which is this. So it should not be put here, but in test files in any case

Reply:
Completely true here. I will make the changes when the DB will be here.